java/res/xml/rowkeys_santali_olchiki1.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+**
+** Copyright 2012, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+
+<merge xmlns:latin="http://schemas.android.com/apk/res-auto">
+
+    <Key
```

Comment:
Can you see once?


On Thu, Jun 2, 2016 at 12:47 PM, Jishnu Mohan <notifications@github.com>
wrote:

> In java/res/xml/rowkeys_santali_olchiki1.xml
> <https://github.com/smc/Indic-Keyboard/pull/69#discussion_r65493933>:
>
> > +** you may not use this file except in compliance with the License.
> > +** You may obtain a copy of the License at
> > +**
> > +**     http://www.apache.org/licenses/LICENSE-2.0
> > +**
> > +** Unless required by applicable law or agreed to in writing, software
> > +** distributed under the License is distributed on an "AS IS" BASIS,
> > +** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +** See the License for the specific language governing permissions and
> > +** limitations under the License.
> > +*/
> > +-->
> > +
> > +<merge xmlns:latin="http://schemas.android.com/apk/res-auto">
> > +
> > +    <Key
>
> are you sure that this file is needed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/smc/Indic-Keyboard/pull/69/files/c2e980bd6b962dd57bca932bf8004e51ef848f7f#r65493933>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI68nMEj7bHKf7N09xk5kfgGqEmWXS05ks5qHoOZgaJpZM4Iryx0>
> .
>